_layouts/default.html

```diff
@@ -25,17 +45,60 @@
             <nav aria-label="breadcrumb" class="d-none d-md-block d-print-none">
 	      <ol class="breadcrumb spb-1">
               {% assign crumbs = page.url | append: "/"  | replace: '//', '/' | split: '/' %}
-              {% for crumb in crumbs offset: 1 %}{% if crumb != "index" %}{% if forloop.last %}
-                <li class="breadcrumb-item active" aria-current="page">
-	          <a href="{{ site.url }}{{ site.baseurl }}{{ page.url }}">{{ page.title }}</a>
-                </li>{% else %}
-                {% if page.url != "/docs/" %}<li class="breadcrumb-item">
-                  <a href="{{ site.url }}{{ site.baseurl }}{% assign crumb_limit = forloop.index | plus: 1 %}{% for crumb in crumbs limit: crumb_limit %}{{ crumb | append: '/' }}{% endfor %}">{% if crumb == "docs" %}Documentation{% else %}{{ crumb | replace:'-',' ' | remove:'.html' | capitalize }}{% endif %}</a>
-                </li>{% endif %}{% endif %}
-                {% endif %}{% endfor %}
+              {% for crumb in crumbs offset: 1 %}
+                {% if crumb != "index" %}
+                  {% if forloop.last %}
+                    <li class="breadcrumb-item active" aria-current="page">
+                      <a href="{{ site.url }}{{ site.baseurl }}{{ page.url }}">{{ page.title }}</a>
+                    </li>
+                  {% else %}
+                    {% if page.url != "/docs/" %}
+                      {% if site.version_params.versioning == true and crumb == currentpageversion or crumb == site.version_params.version_dir %}
+                      <!-- Don't display the version number or directory in the breadcrumb -->
+                      {% else %}
+                      <li class="breadcrumb-item">
+                        <a href="{{ site.url }}{{ site.baseurl }}{% assign crumb_limit = forloop.index | plus: 1 %}{% for crumb in crumbs limit: crumb_limit %}{{ crumb | append: '/' }}{% endfor %}">{% if crumb == "docs" %}Documentation{% else %}{{ crumb | replace:'-',' ' | remove:'.html' | capitalize }}{% endif %}</a>
+                      </li>
+                      {% endif %}
+                    {% endif %}
+                  {% endif %}
+                {% endif %}
+              {% endfor %}
 	      </ol>
            </nav>
-           <div class="td-content">
+           <div class="td-content">        
+            {% if site.version_params.versioning == true %}
+              {% if site.version_params.latest != currentpageversion %}
+              {% capture civersion %}
+                {{- currentpageversion | downcase | strip -}}
+              {% endcapture %}
+              <div class="archive_note">
```

Comment:
I love this little customization! <3 Should this one snippet be put in an include so we can easily extend if needed?

Reply:
I went ahead and created this as versionalert.html in the _includes directory and mentioned in the documentation that you can customise it there.